<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>987</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>631</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>621</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>621</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>621</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Revert</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reselected_table">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>621</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Update table</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_row">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>20</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Add row</string>
   </property>
  </widget>
  <widget class="QPushButton" name="detele_row">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>20</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete row</string>
   </property>
  </widget>
  <widget class="QPushButton" name="show_tables">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>90</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Tables</string>
   </property>
  </widget>
  <widget class="QPushButton" name="export_csv">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>90</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Export CSV</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>20</y>
     <width>631</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="log">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>440</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>410</y>
     <width>64</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Logging</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
